tfx/experimental/templates/taxi/launcher/template_stub_component_launcher.py

```diff
@@ -0,0 +1,40 @@
+# Lint as: python3
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Template stub component launcher for launching stub executors in KFP."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tfx.experimental.pipeline_testing import base_stub_executor
+from tfx.experimental.pipeline_testing import stub_component_launcher
+
+class TemplateStubComponentLauncher(
```

Comment:
I think that 'template' in the name of this class seems odd. Because this class is  not related to template. Most users will not see "template" in the path of this file while using this.

How about calling this just `StubComponentLauncher` or `TaxiStubComponentLauncher`?

Reply:
I think that it would be better to use `get_stub_launcher_class()`.